### PR TITLE
Changed sections

### PR DESCRIPTION
**Pull request by neewy**
Reworked overview section, block description, build details, prettified listings and styles

### CODE REVIEW

File: source/includes/_build.md [hunk 1]
```diff
@@ -15,28 +21,45 @@ https://github.com/hyperledger/iroha /path/to/iroha
...
+After you execute this script, following things happen:
 
-Docker environment will be removed when you logout from the container.
+ 1. It checks if you don't have containers with Iroha already running. If yes — it will reattach you to interactive shell. Assuming if you don't have any, then:
+ 2. The script will download iroha-docker-develop, redis and postgres images. Iroha image contains all development dependencies, and is based on top of ubuntu:16.04.
+ 3. Three containers are created and launched.
+ 4. The user is attached to the interactive environment for development and testing with `iroha` folder mounted from the host machine.
 
-### Build `iroha` and run tests
+<aside class="notice">
```
Comment (luckychess) [inline source/includes/_build.md]: Does this line have any effect on resulting text?

File: source/includes/_build.md [hunk 2]
```diff
@@ -15,28 +21,45 @@ https://github.com/hyperledger/iroha /path/to/iroha
...
+
+> Build
+
+``` bash
 cmake -H. -Bbuild;
 cmake --build build -- -j$(nproc)
 ```
 
-`irohad` and `iroha-cli` binaries will be in `./build/bin` directory.
+<aside class="notice">
```
Comment (luckychess) [inline source/includes/_build.md:47]: May be this it will be better to show this paragraph like a note? It looks like not a really important part of manual.

File: source/includes/_build.md [hunk 3]
```diff
@@ -15,28 +21,45 @@ https://github.com/hyperledger/iroha /path/to/iroha
...
+
+We use CMake to build platform-dependent build files. Run shell commands from "Build" section on the right. Built binaries (`irohad` and `iroha-cli`) will be in `./build/bin` directory.
+
+After you built the project — please run tests to check the operability of the daemon.
 
 ### Add to irohad and iroha-cli to path (optional)
 
 export PATH=/path/to/iroha/build/bin:$PATH
 
+<aside class="notice">
```
Comment (luckychess) [inline source/includes/_build.md]: Same as above.

File: source/includes/_build.md [hunk 4]
```diff
@@ -55,52 +78,43 @@ Execute `run-iroha-dev.sh` again to attach to existing container.
...
 ## Linux or macOS
 
-For Iroha, a provisioning of Postgres and Redis are required. You may launch them on your local machine, or use docker containers, as provided.
-
 > Launching Docker and Postgres in Docker
 
 ``` shell
-docker run --name some-redis -p 6379:6379 -d redis:3.2.8
-docker run --name some-postgres -e POSTGRES_PASSWORD=mysecretpassword -p 5432:5432 -d postgres:9.5
+docker run --name some-redis \
```
Comment (luckychess) [inline source/includes/_build.md]: May be not to split this lines? They are quite short.

File: source/includes/_build.md [hunk 5]
```diff
@@ -55,52 +78,43 @@ Execute `run-iroha-dev.sh` again to attach to existing container.
...
+-p 6379:6379 -d redis:3.2.8
+docker run --name some-postgres \
+-e POSTGRES_USER=iroha \
+-e POSTGRES_PASSWORD=helloworld \
+-p 5432:5432 -d postgres:9.5
 ```
 
-> Linux
+To launch Iroha daemon, running postgres and redis services are required. You may launch them on your local machine, or use docker containers, as provided on the right side.
```
Comment (luckychess) [inline source/includes/_build.md]: What do you mean as a "right side"?

File: source/includes/_build.md [hunk 6]
```diff
@@ -55,52 +78,43 @@ Execute `run-iroha-dev.sh` again to attach to existing container.
...
-git clone https://github.com/hyperledger/iroha -b develop
-cd iroha
-cmake -H. -Bbuild
-cmake --build build -- -j4
-```
+### Linux (debian-based)
 
-> macOS
+To install dependencies, clone, and build the project, please use this code:
+<script src="https://gist.github.com/neewy/39557aa444c3317aeffbdedd9f0f51e2.js"></script>
```
Comment (luckychess) [inline source/includes/_build.md:100]: Looks like tag <script> didn't work (

File: source/includes/_build.md [hunk 7]
```diff
@@ -55,52 +78,43 @@ Execute `run-iroha-dev.sh` again to attach to existing container.
...
-```
 > macOS brew
 
 ``` shell
 brew tap soramitsu/iroha
 brew install iroha
 ```
+
+To install dependencies, clone, and build the project, please use this code:
+<script src="https://gist.github.com/neewy/bc0fea777592a5381aa4ab6e68bfe699.js"></script>
```
Comment (luckychess) [inline source/includes/_build.md:123]: Same issue with <script> tag.

File: source/includes/_overview.md [hunk 8]
```diff
@@ -2,18 +2,27 @@
 
 Welcome to the Hyperledger Iroha API docs!
 
-<iframe width="560" height="315" style="padding-left: 28px" src="https://www.youtube.com/embed/Hf_5j0b6TZM" frameborder="0" allowfullscreen></iframe>
+The documentation describes the endpoints for users to send transactions to peer network, containing one or many commands to perform allowed actions in the system, and also make queries to know the current state. Users are authenticated by their keypair and account id passed as the parameter with their transactions or queries.
```
Comment (luckychess) [inline source/includes/_overview.md]: Looks like documentation describes a bit more than just endpoints how to send data to Iroha. Also I'm not sure note about account id should be here.
May be just move this paragraph after one about elevator pitch?

File: source/includes/_overview.md [hunk 9]
```diff
@@ -2,18 +2,27 @@
...
 
-The API is organized around protobuf format, as Iroha is using gRPC on transport level. Iroha CLI and blockstore works with JSON format to provide developer-friendly experience for community members.  
+## Elevator pitch for Iroha
 
-To try out a basic API request, do the following:
+The system is a distributed ledger platform, capable of storing all history of state-changing actions (called transactions) in immutable linear history (called blockchain). The system is intended to be used in enterprise or between untrusted agents as **private chain platform**, but not as a public chain — key difference is that not everyone is allowed to store the whole history. A subset of users can only query some data (perform a query) if their accounts have required permissions, while others maintain a peer (Iroha is a peer network), which stores history. Reliability and security are provided with cryptographic capabilities and **Byzantine Fault Tolerant** consensus algorithm. The system is targeted to environments, where client applications are very diverse (desktop, various mobile platforms), and peer hardware varies from embedded systems to enterprise-class servers.
```
Comment (luckychess) [inline source/includes/_overview.md]: I'm not sure but I think it's article lost before "private chain platform" words.
Also I didn't get the point of "The system is targeted to environments" sentence.

File: source/includes/_overview.md [hunk 10]
```diff
@@ -2,18 +2,27 @@
...
-2. Run iroha-cli in interactive mode
-3. Select a necessary action to perform
-4. Send formed protobuf via gRPC call
-5. Get a response from the system 
+## Try it yourself
+ 
+To try out a basic API functionality, do the following:
 
+<aside class="notice">
+In case you have troubles understanding some specific terminology used — refer to the glossary.
```
Comment (luckychess) [inline source/includes/_overview.md]: Link to glossary?

File: source/includes/_overview.md [hunk 11]
```diff
@@ -2,18 +2,27 @@
...
+</aside>
+
+1. Run the system (`irohad` daemon) on a single node.
+2. Run iroha-cli in interactive mode. Command Line Interface app is a client, showing the capabilities of the system.
+3. Select a necessary action to perform. As you created an initial configuration in genesis block — use the account from genesis block to send transaction or make a query.
+4. When you fill all the necessary details for commands and formed a transaction; or if you formed a query — you are ready to send it to Iroha peer. Tell the network address and port (by default it is 50001).
```
Comment (luckychess) [inline source/includes/_overview.md]: Default is 50051.

File: source/includes/_run.md [hunk 12]
```diff
@@ -1,30 +1,64 @@
...
+Generate genesis block using iroha-cli (see shell command on the right side).
+
+Iroha-cli tool is used to generate keypairs for every peer, mentioned in the peers list. This is done only for convenience — in production environment please consider using manual or more trusted approach.
+
+> Generating genesis block
 
-2. Generate genesis block using iroha-cli:
 ```bash
-$ iroha-cli --genesis_block --peers_address peers.list
+$ iroha-cli --genesis_block \
```
Comment (luckychess) [inline source/includes/_run.md:38]: Single line will look better =)

File: source/includes/_run.md [hunk 13]
```diff
@@ -1,30 +1,64 @@
...
-2. Generate genesis block using iroha-cli:
 ```bash
-$ iroha-cli --genesis_block --peers_address peers.list
+$ iroha-cli --genesis_block \ 
+--peers_address peers.list
 ```
+Executing this command will result in generation of following files:
+
+* `admin@test.priv` and `admin@test.pub` — keypair for user with admin and asset creator rights
+* `test@test.priv` and `tes@test.pub` — keypair for user with user rights
```
Comment (luckychess) [inline source/includes/_run.md]: Missed 't' at first part of second file name.

File: source/includes/_run.md [hunk 14]
```diff
@@ -1,30 +1,64 @@
...
+* `node0.priv` and `node0.pub` (and other keypairs for each node, starting with 0) — keypair for first node, mentioned in the peers.list
+* `genesis.block` — genesis block itself
+
+<aside class="notice">
+What are those asset creator, admin, user rights? Iroha supports role-based access control, so initial roles for accounts should be created in genesis block. Check permission model section of the documentation to find out more.
```
Comment (luckychess) [inline source/includes/_run.md]: Link to permission model section would be great.

File: source/includes/_run.md [hunk 15]
```diff
@@ -1,30 +1,64 @@
...
 
 Configuration file keeps information about storage credentials and irohad parameters:
 
 | Parameter         | Type    | Meaning                                                                                       |
 |-------------------|---------|-----------------------------------------------------------------------------------------------|
-| block_store_path  | string  | Path to store blocks of committed transactions                                                |
-| torii_port        | integer | Port to access iroha node                                                                     |
-| internal_port     | integer | Port used to communicate ordering service, YAC consensus and block loader for synchronization |
+| block_store_path  | string  | Path to store blocks of committed transactions (flat file storage)                            |
+| torii_port        | integer | Port to access iroha node gRPC (default 50001)                                                |
```
Comment (luckychess) [inline source/includes/_run.md]: 50051

File: source/includes/_run.md [hunk 16]
```diff
@@ -33,20 +67,7 @@ Configuration file keeps information about storage credentials and irohad parame
...
-  "pg_opt" : "host=localhost port=5432 user=postgres password=mysecretpassword",
-  "redis_host" : "localhost",
-  "redis_port" : 6379,
-  "max_proposal_size" : 10,
-  "proposal_delay" : 5000,
-  "vote_delay" : 5000,
-  "load_delay" : 5000
-}
-```
+<script src="https://gist.github.com/neewy/1ebe653de074dac1675f74eb19824b2e.js"></script>
```
Comment (luckychess) [inline source/includes/_run.md:76]: Broken tag again.

**Review by luckychess — CHANGES_REQUESTED** [review]
Sorry for being overpedantic - just wanted to make docs better =)

File: source/includes/_build.md [hunk 17]
```diff
@@ -15,28 +21,45 @@ https://github.com/hyperledger/iroha /path/to/iroha
...
+On macOS $(nproc) variable does not work. Check number of logical cores with sysctl -n hw.ncpu
+</aside>
+
+We use CMake to build platform-dependent build files. Run shell commands from "Build" section on the right. Built binaries (`irohad` and `iroha-cli`) will be in `./build/bin` directory.
+
+After you built the project — please run tests to check the operability of the daemon.
 
 ### Add to irohad and iroha-cli to path (optional)
 
 export PATH=/path/to/iroha/build/bin:$PATH
```
Comment (kamilsa) [inline source/includes/_build.md]: Please, be consistent. On line 24, you demonstrated path starting from `.../iroha/` while here it is `/path/to/iroha`. Use common style for both cases

File: source/includes/_build.md (hunk 5 above)
Comment (kamilsa) [reply source/includes/_build.md]: When this .md file is finished it will look in hyperledger.github.io/iroha-api as two sided page: left one is for text and right one for the code

File: source/includes/_build.md (hunk 6 above)
Comment (kamilsa) [reply source/includes/_build.md:100]: @neewy will it work in slate?

File: source/includes/_build.md (hunk 7 above)
Comment (kamilsa) [reply source/includes/_build.md:123]: Is there a way to see the code inside js file you are pointing here?

File: source/includes/_overview.md (hunk 9 above)
Comment (kamilsa) [reply source/includes/_overview.md]: IMO this paragraph should be right after welcome sentence (line 3).

File: source/includes/_run.md [hunk 18]
```diff
@@ -1,30 +1,64 @@
...
+<b>Important:</b> each peer in the initialized network should have exactly the same genesis block, so create it once and distribute across peers. This block may have <b>any data</b>, as no-one validates it in the peer network.
+
+### Using iroha-cli
+
+#### Create list of peers
+Create peers.list file containing network addresses of peers' internal ports in the network (a single peer address goes on new line. If you want to run Iroha on single peer, it is sufficient to mention only localhost: <br> `$ echo 'localhost:10001' > peers.list`
+
+As an example, for 4 peers the file will look like the following:
+<ul>
+  <li>10.128.13.1:10001</li>
```
Comment (kamilsa) [inline source/includes/_run.md]: As far as I remember first line in peers.list refers to ordering service (internal port), and has port number 10001. Remaining ones correspond to the peers' addresses and have port 50051

File: source/includes/_run.md [hunk 19]
```diff
@@ -1,30 +1,64 @@
...
 ```bash
-$ iroha-cli --genesis_block --peers_address peers.list
+$ iroha-cli --genesis_block \ 
+--peers_address peers.list
 ```
+Executing this command will result in generation of following files:
+
+* `admin@test.priv` and `admin@test.pub` — keypair for user with admin and asset creator rights
+* `test@test.priv` and `tes@test.pub` — keypair for user with user rights
+* `node0.priv` and `node0.pub` (and other keypairs for each node, starting with 0) — keypair for first node, mentioned in the peers.list
```
Comment (kamilsa) [inline source/includes/_run.md]: What means starting with 0?

**Review by kamilsa — CHANGES_REQUESTED** [review]

File: source/includes/_build.md [hunk 20]
```diff
@@ -15,28 +21,45 @@ https://github.com/hyperledger/iroha /path/to/iroha
 
 ### How to run development environment
 
-``` shell
-/path/to/iroha/scripts/run-iroha-dev.sh
-```
+Run the script `run-iroha-dev.sh`, contained in the folder `scripts`: `sh .../iroha/scripts/run-iroha-dev.sh`
 
-You will be attached to interactive environment for development and testing with `iroha` folder mounted from host machine.
+After you execute this script, following things happen:
```
Comment (l4l) [inline source/includes/_build.md:26]:
```
... following things hapen:
1. It checks ...
```
A bit of inconsistency here

File: source/includes/_build.md [hunk 21]
```diff
@@ -15,28 +21,45 @@ https://github.com/hyperledger/iroha /path/to/iroha
...
 
-Docker environment will be removed when you logout from the container.
+ 1. It checks if you don't have containers with Iroha already running. If yes — it will reattach you to interactive shell. Assuming if you don't have any, then:
```
Comment (l4l) [inline source/includes/_build.md]: Maybe rephrase it to
"<s>If yes —</s> It <b>ends up with the</b> <s>re</s>attach<b>ing</b> you to interactive shell **upon succesful completion**. <s>Assuming if you don't have any, then:</s>"
or smth similar?

File: source/includes/_build.md [hunk 22]
```diff
@@ -15,28 +21,45 @@ https://github.com/hyperledger/iroha /path/to/iroha
...
+ 2. The script will download iroha-docker-develop, redis and postgres images. Iroha image contains all development dependencies, and is based on top of ubuntu:16.04.
+ 3. Three containers are created and launched.
+ 4. The user is attached to the interactive environment for development and testing with `iroha` folder mounted from the host machine.
```
Comment (l4l) [inline source/includes/_build.md]: Mention that iroha mounted to `/opt/iroha`

File: source/includes/_build.md [hunk 23]
```diff
@@ -15,28 +21,45 @@ https://github.com/hyperledger/iroha /path/to/iroha
...
+> Build
+
+``` bash
 cmake -H. -Bbuild;
 cmake --build build -- -j$(nproc)
 ```
 
-`irohad` and `iroha-cli` binaries will be in `./build/bin` directory.
+<aside class="notice">
+On macOS $(nproc) variable does not work. Check number of logical cores with sysctl -n hw.ncpu
```
Comment (l4l) [inline source/includes/_build.md]: Surrond sysctl command with ` quotes

File: source/includes/_build.md (hunk 4 above)
Comment (l4l) [reply source/includes/_build.md]: Agree, at least with the first. Allso some nice aligment can be used, e.g:
```
docker run --name some-redis -p 6379:6379\
                             -d redis:3.2.8
docker run --name some-postgres -e POSTGRES_USER=iroha \
                                -e POSTGRES_PASSWORD=helloworld \
                                -p 5432:5432 -d postgres:9.5
```

Comment (l4l) [reply source/includes/_build.md]: Also versions can be removed

File: source/includes/_build.md [hunk 24]
```diff
@@ -55,52 +78,43 @@ Execute `run-iroha-dev.sh` again to attach to existing container.
...
-> macOS
+To install dependencies, clone, and build the project, please use this code:
+<script src="https://gist.github.com/neewy/39557aa444c3317aeffbdedd9f0f51e2.js"></script>
+
+#### Outdated dependencies 
+
+In case that some dependencies are outdated in apt, you are advised to use following sources to install dependencies:
+
+##### Boost
+To install Boost libraries (`libboost-all-dev`), use [current release](http://www.boost.org/users/download/) from Boost webpage.
```
Comment (l4l) [inline source/includes/_build.md]: [There're exist](https://packages.debian.org/sid/libboost-all-dev) repos for newer one, that can be more practical

File: source/includes/_build.md [hunk 25]
```diff
@@ -55,52 +78,43 @@ Execute `run-iroha-dev.sh` again to attach to existing container.
...
+Exact dependencies are system and filesystem, so use `./bootstrap.sh --with-libraries=system,filesystem` when you are building the project.
```
Comment (l4l) [inline source/includes/_build.md]: s/Exact/The only/

File: source/includes/_run.md [hunk 26]
```diff
@@ -1,30 +1,64 @@
...
+# Run the daemon (irohad) 
 
-## Generate genesis block
+## Create genesis block
 
-1. Create peers.list file containing the peers' addresses. For the local node suffice to add only single peer:
-```bash
-$ echo 'localhost:10001' > peers.list
-```
+A genesis block is the first block in the chain. <br>
```
Comment (l4l) [inline source/includes/_run.md]: `<br>` D:
Just use an empty line, don't merge paragraphs

File: source/includes/_run.md [hunk 27]
```diff
@@ -1,30 +1,64 @@
...
+<b>Important:</b> each peer in the initialized network should have exactly the same genesis block, so create it once and distribute across peers. This block may have <b>any data</b>, as no-one validates it in the peer network.
```
Comment (l4l) [inline source/includes/_run.md]: `<b>any data</b>` -> `<b>any</b> data`

File: source/includes/_run.md [hunk 28]
```diff
@@ -56,10 +77,8 @@ To launch irohad daemon, following parameters must be passed:
 |---------------|----------------------------------------------------------------------------------------------|
 | config        | configuration file, containing postgres, and redis connection, and values to tune the system |
 | genesis_block | initial block in the ledger                                                                  |
-| keypair_name  | private and public key file names without .priv or .pub extension. Used by peer to sign the blocks                             |
+| keypair_name  | private and public key file names without extension (.priv or .pub). Used by peer to sign the blocks |
```
Comment (l4l) [inline source/includes/_run.md]: just `file extension` is enough i think

**Review by l4l — CHANGES_REQUESTED** [review]
There's missing cmake flags in the doc:
```
-- -DTESTING=ON
-- -DBENCHMARKING=OFF
-- -DFUZZING=OFF
-- -DCOVERAGE=OFF
-- -DPACKAGE_ZIP=OFF
-- -DPACKAGE_TGZ=OFF
-- -DPACKAGE_RPM=OFF
-- -DPACKAGE_DEB=OFF
-- -DENABLE_LIBS_PACKAGING=ON
-- -DSWIG_PYTHON=ON
-- -DSWIG_JAVA=ON
```

I think they should be listed

File: source/includes/_build.md (hunk 1 above)
Comment (neewy) [reply source/includes/_build.md]: Yes, https://github.com/lord/slate/wiki/Markdown-Syntax

File: source/includes/_build.md (hunk 20 above)
Comment (neewy) [reply source/includes/_build.md:26]: script - it checks? What is the inconsistency?

File: source/includes/_build.md (hunk 21 above)
Comment (neewy) [reply source/includes/_build.md]: Ok, but I see no problem at the moment

File: source/includes/_build.md (hunk 22 above)
Comment (neewy) [reply source/includes/_build.md]: ok

File: source/includes/_build.md (hunk 2 above)
Comment (neewy) [reply source/includes/_build.md:47]: It looks like a note

File: source/includes/_build.md (hunk 23 above)
Comment (neewy) [reply source/includes/_build.md]: ok

File: source/includes/_build.md (hunk 17 above)
Comment (neewy) [reply source/includes/_build.md]: ok

File: source/includes/_build.md (hunk 3 above)
Comment (neewy) [reply source/includes/_build.md]: It looks like note

File: source/includes/_build.md (hunk 4 above)
Comment (neewy) [reply source/includes/_build.md]: I expanded the size of text section to 35% for better readability and this is the reason of splitting it

Comment (neewy) [reply source/includes/_build.md]: screen shot 2017-12-16 at 16 22 36

File: source/includes/_build.md (hunk 1 above)
Comment (neewy) [reply source/includes/_build.md]: screen shot 2017-12-16 at 16 22 29

File: source/includes/_build.md (hunk 20 above)
Comment (neewy) [reply source/includes/_build.md:26]: I will swap it with the script

File: source/includes/_build.md (hunk 21 above)
Comment (neewy) [reply source/includes/_build.md]: done

File: source/includes/_build.md (hunk 23 above)
Comment (neewy) [reply source/includes/_build.md]: done

File: source/includes/_build.md (hunk 17 above)
Comment (neewy) [reply source/includes/_build.md]: changed

File: source/includes/_build.md (hunk 3 above)
Comment (neewy) [reply source/includes/_build.md]: screen shot 2017-12-16 at 16 31 59

File: source/includes/_build.md (hunk 4 above)
Comment (neewy) [reply source/includes/_build.md]: Now it look like this, I also changed the password to work with tests and example of config:
screen shot 2017-12-16 at 16 35 24

File: source/includes/_build.md (hunk 5 above)
Comment (neewy) [reply source/includes/_build.md]: Let me show:
screen shot 2017-12-16 at 16 46 34

File: source/includes/_build.md (hunk 6 above)
Comment (neewy) [reply source/includes/_build.md:100]: screen shot 2017-12-16 at 16 50 26

File: source/includes/_build.md (hunk 24 above)
Comment (neewy) [reply source/includes/_build.md]: I will suggest to add repos

File: source/includes/_build.md (hunk 25 above)
Comment (neewy) [reply source/includes/_build.md]: ok

File: source/includes/_build.md (hunk 7 above)
Comment (neewy) [reply source/includes/_build.md:123]: yes, it shows fine

Comment (neewy) [reply source/includes/_build.md:123]: screen shot 2017-12-16 at 16 50 14

File: source/includes/_overview.md (hunk 8 above)
Comment (neewy) [reply source/includes/_overview.md]: The primary purpose of this website is to tell about endpoints and how to build the system. Later, we will migrate every information not relevant to endpoints to RTD. I will paraphrase and remove id in account id.

File: source/includes/_overview.md (hunk 9 above)
Comment (neewy) [reply source/includes/_overview.md]: Okay, I will swap them

Comment (neewy) [reply source/includes/_overview.md]: @luckychess about env -- the sentence is about where the system is used or applied, and the range of application is very broad and diverse. This tells about portability and compatibility of the system

File: source/includes/_overview.md (hunk 10 above)
Comment (neewy) [reply source/includes/_overview.md]: Not yet, in separate PR

File: source/includes/_overview.md (hunk 11 above)
Comment (neewy) [reply source/includes/_overview.md]: ok, sure

File: source/includes/_overview.md (hunk 10 above)
Comment (neewy) [reply source/includes/_overview.md]: I will create a link for now

File: source/includes/_run.md (hunk 26 above)
Comment (neewy) [reply source/includes/_run.md]: ok

File: source/includes/_run.md (hunk 18 above)
Comment (neewy) [reply source/includes/_run.md]: okay...

File: source/includes/_run.md (hunk 12 above)
Comment (neewy) [reply source/includes/_run.md:38]: It won't fit, let's leave it as it is for now
screen shot 2017-12-16 at 17 02 10

File: source/includes/_run.md (hunk 13 above)
Comment (neewy) [reply source/includes/_run.md]: ok

File: source/includes/_run.md (hunk 28 above)
Comment (neewy) [reply source/includes/_run.md]: ok

File: source/includes/_run.md (hunk 16 above)
Comment (neewy) [reply source/includes/_run.md:76]: screen shot 2017-12-16 at 17 04 40

File: source/includes/_run.md (hunk 15 above)
Comment (neewy) [reply source/includes/_run.md]: ok

File: source/includes/_run.md (hunk 14 above)
Comment (neewy) [reply source/includes/_run.md]: Link is fine, but the page is in separate PR, okay?

File: source/includes/_build.md (hunk 3 above)
Comment (luckychess) [reply source/includes/_build.md]: Oh, I see. I opened this page throw github which ignores such tags.

**Comment by neewy** [comment]
Thanks everyone for the review! I really hope we can make them better before the release.
In order to check them how they look please run https://github.com/lord/slate/wiki/Installing-Slate#getting-set-up

File: source/includes/_build.md [hunk 29]
```diff
@@ -25,10 +25,10 @@ Run the script `run-iroha-dev.sh`, contained in the folder `scripts`: `sh .../ir
 
 After you execute this script, following things happen:
 
- 1. It checks if you don't have containers with Iroha already running. If yes — it will reattach you to interactive shell. Assuming if you don't have any, then:
+ 1. The script if you don't have containers with Iroha already running. It ends up with reattaching you to interactive shell upon succesful completion.
```
Comment (luckychess) [inline source/includes/_build.md]: Return "it checks" back plz :)

**Review by luckychess — APPROVED** [review]

File: source/includes/_build.md (hunk 29 above)
Comment (neewy) [reply source/includes/_build.md]: あああ

Comment (neewy) [reply source/includes/_build.md]: done

File: source/includes/_build.md (hunk 20 above)
Comment (l4l) [reply source/includes/_build.md:26]: I mean not really clear who is "it"

**Review by l4l — APPROVED** [review]
